app/controllers/training/questions_controller.rb

```diff
@@ -50,6 +52,11 @@ def track_confidence_start?
       content.first_confidence? && confidence_start_untracked?
     end
 
+    # @return [Boolean]
+    def track_feedback_start?
+      content.feedback_question? || content.opinion_intro?
```

Comment:
Why are there two pages here? Will this result in two events when there should be only one?